cmake/FPrime.cmake

```diff
@@ -160,6 +160,9 @@ function(fprime_setup_included_code)
     setup_global_targets()
     # For BUILD_TESTING builds then set up libraries that support testing
     if (BUILD_TESTING AND NOT DEFINED FPRIME_SUB_BUILD_TARGETS)
+        if (NOT EXISTS "${FPRIME_FRAMEWORK_PATH}/googletest/CMakeLists.txt")
+            message(FATAL_ERROR "googletest submodule not initialized or corrupted. Please run `git submodule update --init`.")
```

Comment:
We should include `--recursive` because projects users would need this flag if running from within project.